pages/accountLists/[accountListId]/reports/expectedMonthlyTotal.page.tsx

```diff
@@ -2,29 +2,117 @@ import React, { ReactElement } from 'react';
 import Head from 'next/head';
 import { useTranslation } from 'react-i18next';
 import { Box } from '@material-ui/core';
-import { ExpectedMonthlyTotalReportHeader } from '../../../../src/components/Reports/ExpectedMonthlyTotalReport/ExpectedMonthlyTotalReportHeader';
-import { ExpectedMonthlyTotalReportTable } from '../../../../src/components/Reports/ExpectedMonthlyTotalReport/ExpectedMonthlyTotalReportTable';
+import { ExpectedMonthlyTotalReport } from '../../../../src/components/Reports/ExpectedMonthlyTotalReport/ExpectedMonthlyTotalReport';
+import { ExpectedMonthlyTotalReportHeader } from '../../../../src/components/Reports/ExpectedMonthlyTotalReport/Header/ExpectedMonthlyTotalReportHeader';
 import Loading from '../../../../src/components/Loading';
 import { useAccountListId } from '../../../../src/hooks/useAccountListId';
 
 const ExpectedMonthlyTotalReportPage = (): ReactElement => {
   const { t } = useTranslation();
   const accountListId = useAccountListId();
 
+  function createData(
+    contact: string,
+    contactId: string,
+    status: string,
+    commitment: string,
+    frequency: string,
+    converted: string,
+    currency: string,
+    donation: string,
+  ) {
+    return {
+      contact,
+      contactId,
+      status,
+      commitment,
+      frequency,
+      converted,
+      currency,
+      donation,
+    };
+  }
+
+  const rows = [
+    createData(
+      'Adriano, Selinda',
+      'abc',
+      'Partner - Financial',
+      '50',
+      'Monthly',
+      '50',
+      'CAD',
+      '50',
+    ),
+    createData(
+      'Adriano, Selinda',
+      'abc',
+      'Partner - Financial',
+      '50',
+      'Monthly',
+      '50',
+      'CAD',
+      '50',
+    ),
+    createData(
+      'Adriano, Selinda',
+      'abc',
+      'Partner - Financial',
+      '50',
+      'Monthly',
+      '50',
+      'CAD',
+      '50',
+    ),
+    createData(
+      'Adriano, Selinda',
+      'abc',
+      'Partner - Financial',
+      '50',
+      'Monthly',
+      '50',
+      'CAD',
+      '50',
+    ),
+    createData(
+      'Adriano, Selinda',
+      'abc',
+      'Partner - Financial',
+      '50',
+      'Monthly',
+      '50',
+      'CAD',
+      '50',
+    ),
+    createData(
+      'Adriano, Selinda',
+      'abc',
+      'Partner - Financial',
+      '50',
+      'Monthly',
+      '50',
+      'CAD',
+      '50',
+    ),
+  ];
+
   return (
     <>
       <Head>
         <title>
           MPDX | {t('Reports')} | {t('Expect Monthly Total')}
         </title>
       </Head>
-      {accountListId ? (
+      {isReady && accountListId ? (
+        <ExpectedMonthlyTotalReport
+          accountListId={accountListId}
+          data={rows}
```

Comment:
This is throwing an error... I think because the `Contact` type is now expecting to find the property `name`.  In your `CreateData()` function above, can you change `contact` to `name`?